scripts/_utils.ts

```
@@ -18,3 +20,48 @@ export async function assertFileExists(path: string) {
     throw Error(`File ${path} does not exist`)
   }
 }
+
+export type Task = {
+  name: string
+  callable: () => Promise<string | void>
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)